pages/order.tsx

```diff
@@ -0,0 +1,715 @@
+                      content={
+                        <FormattedMessage defaultMessage="External reference code for this order. This is usually a reference number from the contributor accounting system." />
+                      }
+                      containerCursor="default"
+                    >
+                      <FormattedMessage id="Fields.PONumber" defaultMessage="PO Number" />
+                    </StyledTooltip>
+                    {`#${order.pendingContributionData.ponumber}`}
+                  </OrderDetails>
+                )}
+                {order.pendingContributionData?.expectedAt && (
+                  <OrderDetails>
+                    <FormattedMessage defaultMessage="Expected" />
+                    {isOverdue ? (
+                      <OverdueTag>
+                        <DateTime
+                          value={order.pendingContributionData.expectedAt}
+                          dateStyle={'medium'}
+                          timeStyle={undefined}
+                        />
+                        <Span textTransform="uppercase" fontWeight="bold" letterSpacing="0.06em">
+                          &nbsp;
+                          <FormattedMessage defaultMessage="Overdue" />
+                        </Span>
+                      </OverdueTag>
+                    ) : (
+                      <DateTime
+                        value={order.pendingContributionData.expectedAt}
+                        dateStyle={'medium'}
+                        timeStyle={undefined}
+                      />
+                    )}
+                  </OrderDetails>
+                )}
+                {order.paymentMethod ? (
+                  <OrderDetails>
+                    <FormattedMessage id="PaidWith" defaultMessage="Paid With" />
+                    <PaymentMethodTypeWithIcon type={order.paymentMethod?.type} iconSize={16} />
+                  </OrderDetails>
+                ) : (
+                  <OrderDetails>
+                    <FormattedMessage id="paymentmethod.label" defaultMessage="Payment Method" />
+                    {i18nPaymentMethodProviderType(
+                      intl,
+                      order.pendingContributionData?.paymentMethod || 'BANK_TRANSFER',
+                    )}
+                  </OrderDetails>
+                )}
+
+                {!isEmpty(order.pendingContributionData?.fromAccountInfo) && (
+                  <OrderDetails>
+                    <FormattedMessage id="Contact" defaultMessage="Contact" />
+                    <span>
+                      {order.pendingContributionData.fromAccountInfo.name} (
+                      <StyledLink href={`mailto:${order.pendingContributionData.fromAccountInfo.email}`} openInNewTab>
+                        {order.pendingContributionData.fromAccountInfo.email}
+                      </StyledLink>
+                      )
+                    </span>
+                  </OrderDetails>
+                )}
```

Comment:
Fyi I have a local case of an order where `fromAccountInfo` is set but doesn't return any email. Not sure if that can happen in prod

![image](https://user-images.githubusercontent.com/1556356/220372386-a7c34d77-6f06-4406-8811-35f6447f955e.png)